core/domain/question_domain.py

```diff
@@ -2157,7 +2161,8 @@ def to_dict(self) -> QuestionSummaryDict:
             'interaction_id': self.interaction_id,
             'last_updated_msec': utils.get_time_in_millisecs(self.last_updated),
             'created_on_msec': utils.get_time_in_millisecs(self.created_on),
-            'misconception_ids': self.misconception_ids
+            'misconception_ids': self.misconception_ids,
+            'version': self.version
```

Comment:
Should we need to add validation for this property to be a non-negative integer (in the validation method)?

Reply:
Yes makes sense. I will do that.